collect_app/src/androidTest/java/org/odk/collect/android/regression/FillBlankFormTest.java

```diff
@@ -606,11 +607,13 @@ public void missingFileMessage_shouldBeDisplayedIfExternalFIleIsMissing() {
                 .copyForm("select_one_external.xml")
                 .startBlankForm("cascading select test")
                 .clickOnText("Texas")
-                .swipeToNextQuestion()
+                .swipeToNextQuestion("-")
                 .assertText("File: " + formsDirPath + "/select_one_external-media/itemsets.csv is missing.")
-                .swipeToNextQuestion()
+                .swipeToNextQuestion("-")
                 .assertText("File: " + formsDirPath + "/select_one_external-media/itemsets.csv is missing.")
-                .swipeToNextQuestion(3)
+                .swipeToNextQuestion("-")
```

Comment:
This form should be fixed. Please do that as a part of this pull request. I think it's a form from https://xlsform.org/en/#cascading-selects or very similar so questions should be like:
`Select state`
`Select county`
`Select city`

Reply:
I'm not sure but I think it's correct behaviour, because test is about missing .csv file. In case if there is missing .csv file there is "-" instead of question and red text "File xyz.csv is missing" under it.

Reply:
Ok I've fixed this.